### DISCUSSION TITLE
建议在问卷把学校和校区分开，按二级目录筛选校区

### DISCUSSION THREAD

**Discussion by icy-blue** [CollegesChat/university-information#69] [Enhancement]
首先向各maintainer致以最高的谢意，我当年怎么就没这么好的东西呜呜呜~

--------------------------------

目前，山东大学在项目中的命名已有——

- 山东大学(威海)
- 山东大学
- 山东大学（兴隆山校区）
- 山东大学（威海校区）
- 山东大学（威海）
- 山东大学（济南）
- 山东大学（青岛校区）
- 山东大学（青岛）

建议保留 济南 青岛 威海 三个校区

文档修改白天再改吧 不过问卷不改好像永远改不完的亚子2333

![img](https://user-images.githubusercontent.com/29377957/123553329-18c7e680-d7ad-11eb-9bd1-e34df8a440ba.png)

**Comment by slanterns** [comment]
感觉校名相关问题暂时可以靠维护一个 alias list 来合并成一个（？）
